light_the_torch/_cb.py

```diff
@@ -176,6 +178,8 @@ def _detect_nvidia_driver_version() -> Optional[Version]:
     "Windows": {
         # Table 2 from
         # https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html
+        Version("12.2"): Version("525.41"),
+        Version("12.1"): Version("527.41"),
         Version("12.0"): Version("527.41"),
```

Comment:
@pmeier Per the latest version of the webpage https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html
```suggestion
        Version("12.6"): Version("528.33"),
        Version("12.5"): Version("528.33"),
        Version("12.4"): Version("528.33"),
        Version("12.3"): Version("528.33"),
        Version("12.2"): Version("528.33"),
        Version("12.1"): Version("528.33"),
        Version("12.0"): Version("528.33"),
```